docusaurus/docs/backends/04-livenet.md

```diff
@@ -0,0 +1,189 @@
+    println!("Token address: {}", token.address().to_string());
+
+    // Uncomment to load existing contract.
+    // let address = "hash-d26fcbd2106e37be975d2045c580334a6d7b9d0a241c2358a4db970dfd516945";
+    // let address = Address::from_str(address).unwrap();
+    // We use the Livenet-specific `load` method to load the contract
+    // that is already deployed.
+    // let mut token = Erc20Deployer::load(env, address);
+
+    // Non-mutable calls are free! Neat, huh? More on that later.
+    println!("Token name: {}", token.name());
+
+    // The next call is mutable, but the cost is lower that the deployment,
+    // so we change the amount of gas
+    env.set_gas(3_000_000_000u64);
+    token.transfer(recipient, U256::from(1000));
+
+    println!("Owner's balance: {:?}", token.balance_of(owner));
+    println!("Recipient's balance: {:?}", token.balance_of(recipient));
+}
+```
+
+:::note
+The above example is a rust binary, not a test. Note that it is also added as a section of the
+`Cargo.toml` file:
+```toml
+[bin]
+name = "erc20_on_livenet"
+path = "src/bin/erc20_on_livenet.rs"
+required-features = ["livenet"]
+test = false
+```
+:::
+
+## Usage
+
+To run the above code, we simply need to run the binary with the `livenet` feature enabled:
+
+```bash
+cargo run --bin erc20_on_livenet --features=livenet
+```
+
+A part of a sample output should look like this:
+
+```bash
+...
+💁  INFO : Calling "hash-d26fcbd210..." with entrypoint "transfer".
+🙄  WAIT : Waiting 15s for "65b1a5d21...".
+🙄  WAIT : Waiting 15s for "65b1a5d21...".
+💁  INFO : Deploy "65b1a5d21..." successfully executed.
+Owner's balance: 4004
+Recipient's balance: 4000
+```
+Those logs are a result of the last 4 lines of the above listing.
+Each deployment or a call to the blockchain will be noted and will take some time to execute.
+We can see that the `transfer` call took over 15 seconds to execute. But calling balance_of was nearly instant
+and cost us nothing. How it is possible?
+
+:::info
+You can see the deployment on http://cspr.live/ - the transfer from the example
+can be seen [here](https://integration.cspr.live/deploy/65b1a5d21174a62c675f89683aba995c453b942c705b404a1f8bbf6f0f6de32a).
+:::
+
+## How Livenet backend works
+All calls of entrypoints executed on a Casper blockchain cost gas - even if they do not change the state.
+It is possible however to query the state of the blockchain for free.
+
+This principle is used in the Livenet backend - all calls that do not change the state of the blockchain are really executed offline - the only thing that is requested from the
+node is the current state. This is why the `balance_of` call was almost instant and free.
+
+Basically, if the entrypoint function is not mutable or does not make an external call, it is executed offline and
+node is used for the state query only.
+
+Take note, that even if the function is not mutable, but the contract is not deployed or loaded, the Livenet backend will 
```

Comment:
If I were a newcomer I could imagine what is a "deploy", but what the heck is "loading" a contract. Somewhere in the `backends` section the idea of loaders should be explained.

Reply:
I've rewritten this section and added another one to cross calls doc.